### DISCUSSION TITLE
Text box with multiple bound pieces of text?

### DISCUSSION THREAD

**Discussion by crener** [AvaloniaUI/Avalonia#5484] [Q&A] [answered]
I want to have a `Textbox`, `TextBlock`, or `Label` with multiple bindings concatenated to form a new string in the UI. In WPF I'd usually go with `TextBlock` and `Run` but that doesn't seem to work in Avalonia. 

What is the recommended way to build UI specific text from multiple sources in Avalonia?

**Comment by kekekeks** [comment, marked as answer]
For now you can use MultiBinding with a multi value converter. `<Run>` is likely to land this year.

**Reply by crener** [reply]
I look forward to seeing it in a future build :)
